docs/docs-content/vm-management/create-manage-vm/advanced-topics/migrate-vm-kubevirt.md

```diff
@@ -80,6 +80,64 @@ from VMware vSphere to Palette VMO.
   - The Palette CLI must have access to both the VMO cluster and the machines to be migrated.
 - The kubectl command-line tool should also be installed. Refer to the
   [kubectl installation](https://kubernetes.io/docs/tasks/tools/install-kubectl/) guide to learn more.
+- We recommend providing a VMware Virtual Disk Development Kit (VDDK) image for the migration. This will significantly
+  speed up the migration.
+
+  - The VDDK image must be built and uploaded to your image registry before starting the migration. Refer to the
+    [Red Hat Documentation](https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.6/html/installing_and_using_the_migration_toolkit_for_virtualization/prerequisites_mtv#creating-vddk-image_mtv)
+    for guidance.
+  - The host with the Palette CLI setup for migration must have access to your image registry.
```

Comment:
```suggestion
  - The migration host must have access to your image registry.
```

Reply:
Thanks, I was trying to think of a better way to write this 😀